src/test/java/hudson/plugins/batch_task/BatchRunTest.java

```diff
@@ -0,0 +1,36 @@
+package hudson.plugins.batch_task;
+
+import org.junit.Rule;
+
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import hudson.model.FreeStyleProject;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class BatchRunTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void testBasic() throws Exception {
+        FreeStyleProject freeStyleProject = r.createFreeStyleProject("tasker");
+        BatchTask task = new BatchTask("test", "echo hello\n");
+        freeStyleProject.addProperty(new BatchTaskProperty(task));
+        HtmlPage page = r.createWebClient().getPage(freeStyleProject, "batchTasks/task/test/execute");
+        String path = page.getWebResponse().getWebRequest().getUrl().getPath();
+        assertTrue("should redirect to noBuilds page: " + path, path.endsWith("/noBuild"));
```

Comment:
Does this test do the same as the one originally written in Groovy? Specially the first part up until the first `assertTrue`